website/content/docs/setup/quickstart/index.md

```diff
@@ -3,6 +3,55 @@ title: Quick Start
 weight: 10
 ---
 
-This guide lets you quickly evaluate {{< name >}}. For a custom setup follow the [Installation instructions](/docs/setup/install).
- 
+This guide lets you quickly evaluate {{< name >}} using the builtin sample connectors and data access module. Instaling {{< name >}} control plane includes: 
+- Installing Hashicorp Vault and connector as the credentials manager 
+- Installing ODPi Egeria lab and connectors as the data catalog
+- Installing Open Policy Agent (OPA) and connector as the policy manager.
+- Arrow-Flight data access module for reading Parquet and CSV datasets
+
+For a custom setup follow the [Installation instructions](/docs/setup/install)
```

Comment:
These are not the all considered control plane. The quick evaluation is for setting a complete environment including third parties. I would suggest to rephrase.

```suggestion
This guide lets you quickly evaluate {{< name >}}. For a custom setup follow the [Installation instructions](/docs/setup/install).

By the end of this guide you will have the following installed to your Kubernetes cluster:
- The control plane of {{< name >}}
- Hashicorp Vault and connector as the credentials manager 
- ODPi Egeria lab and connectors as the data catalog
- Open Policy Agent (OPA) and connector as the policy manager.
- Arrow-Flight data access module for reading Parquet and CSV datasets